### PR TITLE
hotel-service-2 rate details and rate change

### PR DESCRIPTION
**Pull request by henry-yang-sap**

### CODE REVIEW

**Comment by jonikamelcher** [comment]
Hi @henry-yang-sap. Can you please remove this from the API Reference sidebar? The tech writer team will add it to the navigation after we have completed our review. Thank you.

**Comment by henry-yang-sap** [comment]
> Hi @henry-yang-sap. Can you please remove this from the API Reference sidebar? The tech writer team will add it to the navigation after we have completed our review. Thank you.

Hi @jonikamelcher , I have removed the sidebar change. Thanks!

File: src/api-reference/direct-connects/hotel-service-2/Rate-details.markdown [hunk 1]
```diff
@@ -0,0 +1,500 @@
...
+|----------|-----------|-------------|
+|`RoomStay`|`complex`|**Required** Details on the room stay including time span of this room stay, and financial information related to the room stay, including guarantee, deposit, payment, and cancellation penalties. A room stay represents one (1) hotel.|
+
+#### <a name="room-stay"></a>RoomStay
+
+For a description of the relationship between the `RoomID` and `RatePlanID` refer to "Relationship between RoomID and RatePlanID".
+
+|Name|Type|Description|
+|-------------------|--------------|-------------|
+|`RoomTypes`|`complex`|**Required** Details on the room stay including guest counts, time span of this room stay, pointers to res guests, guest memberships, comments, and special requests pertaining to this particular room stay. Financial information related to the room stay, including guarantee, deposit, payment, and cancellation penalties.|
```
Comment (jonikamelcher) [inline src/api-reference/direct-connects/hotel-service-2/Rate-details.markdown:285]: Hello @henry-yang-sap . Can you elaborate on what "pointers to res guests" refers to?

Comment (henry-yang-sap) [reply src/api-reference/direct-connects/hotel-service-2/Rate-details.markdown:285]: > Hello @henry-yang-sap . Can you elaborate on what "pointers to res guests" refers to?

I think it means "tips for the guest." It was copied from src/api-reference/direct-connects/hotel-service-2/Availability.markdown

Comment (jonikamelcher) [reply src/api-reference/direct-connects/hotel-service-2/Rate-details.markdown:285]: Perfect. Thank you.

**Comment by jonikamelcher** [comment]
:shipit:

**Pull request merged**